notebooks/MAST/HSC/CASJOBS/HCV_CASJOBS/HCV_casjobs_demo.ipynb

```diff
@@ -0,0 +1,995 @@
+    "   m.AutoClass, m.ExpertClass, m.NumFilters,\n",
+    "   f.Filter, f.FilterDetFlag, f.VarQualFlag, f.NumLC, \n",
+    "   f.MeanMag, f.MeanCorrMag, f.MAD, f.Chi2\n",
+    "from HCVmatch m\n",
+    "join HCVfilter f on m.MatchID=f.MatchID\n",
+    "join Groups g on m.GroupID=g.GroupID\n",
+    "where f.VarQualFlag='AAAAA'\n",
+    "order by f.MAD desc\n",
+    "\"\"\"\n",
+    "\n",
+    "t0 = time.time()\n",
+    "tab = jobs.quick(query, task_name=\"HCV demo\", context=HSCContext)\n",
+    "\n",
+    "print(f\"Completed in {(time.time()-t0):.1f} sec\")\n",
+    "\n",
+    "# clean up the output format\n",
+    "tab['MeanMag'].format = \"{:.3f}\"\n",
+    "tab['MeanCorrMag'].format = \"{:.3f}\"\n",
+    "tab['MAD'].format = \"{:.4f}\"\n",
+    "tab['Chi2'].format = \"{:.4f}\"\n",
+    "tab['RA'].format = \"{:.6f}\"\n",
+    "tab['Dec'].format = \"{:.6f}\"\n",
+    "\n",
+    "print(f\"MatchID {tab['MatchID'][0]} in group '{tab['TargetName'][0]}' has largest MAD value = {tab['MAD'][0]:.2f}\")\n",
+    "tab"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Get and plot the light curve."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "matchid = tab['MatchID'][0]\n",
+    "mfilter = tab['Filter'][0]\n",
+    "\n",
+    "jobs = mastcasjobs.MastCasJobs(context=HSCContext)\n",
+    "t0 = time.time()\n",
+    "\n",
+    "# get light curves for F606W and F814W\n",
+    "lc = jobs.quick(\"\"\"select * from HCVdetailed\n",
+    "where MatchID={} and Filter='{}'\n",
+    "\"\"\".format(matchid, mfilter), task_name=\"HCV demo\")\n",
+    "print(f\"{(time.time()-t0):.1f} sec: retrieved {len(lc)} {mfilter} measurements\")\n",
+    "\n",
+    "plt.figure(1, (15, 10))\n",
+    "\n",
+    "x = lc['MJD']\n",
+    "y = lc['CorrMag']\n",
+    "e = lc['MagErr']\n",
+    "plt.errorbar(x, y, yerr=e, fmt='ob', ecolor='k', elinewidth=1, markersize=8, label=mfilter)\n",
+    "\n",
+    "plt.gca().invert_yaxis()\n",
+    "plt.xlabel('MJD [days]')\n",
+    "plt.ylabel('magnitude')\n",
+    "plt.legend(loc='best', title=f\"MatchID: {matchid} in {tab['TargetName'][0]} MAD={tab['MAD'][0]:.2f}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Extract cutout images for the entire light curve (since it does not have many points)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# sort images in MJD order\n",
+    "ind = np.argsort(lc['MJD'])\n",
+    "\n",
+    "# we plot zoomed-in and zoomed-out views side-by-side for each selected image\n",
+    "nim = len(ind)*2\n",
+    "ncols = 2 # images per row\n",
+    "nrows = (nim+ncols-1)//ncols\n",
+    "\n",
+    "imsize1 = 19\n",
+    "imsize2 = 101\n",
+    "mra = tab['RA'][0]\n",
+    "mdec = tab['Dec'][0]\n",
+    "\n",
+    "plt.rcParams.update({\"font.size\": 14})\n",
```

Comment:
Should probably store what it was before and restore it after changing it here? (Or maybe just leave it at 16 since I'm not clear why it is changed anyway

Reply:
It can probably be deleted and just be 16 like everything else

Reply:
`plt.rcParams.update({'font.size': 16})` or other values, are a cosmetic change. It might be better to remove for the default.

Reply:
Deleted